scripts/helper.js

```diff
@@ -711,8 +711,10 @@ var permissions = [{
 }, {
   target: `${accounts.history.account}@execute`,
   action: 'cleanptrxs'
-}
-]
+}, {
```

Comment:
This is the same as previously proposals right, we can add to reputation... maybe this permission can be limited to addrep. 

Although I guess in the end all contracts will be under the same permissions anyway...

Reply:
ok, I will fix it